src/modules/timer.rs

```diff
@@ -0,0 +1,83 @@
+use crate::config::TableExt;
+use ansi_term::Color;
+
+use super::{Context, Module};
+
+/// Outputs the time it took the last command to execute
+///
+/// Will only print if last command took more than two seconds to execute.
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("timer")?;
+    let arguments = &context.arguments;
+    let elapsed = arguments
+        .value_of("elapsed_time")
+        .unwrap_or("x") // not a time
+        .parse::<u64>()
+        .ok()?;
+
+    // Question: Is this a good series of calls to use? Is there a cleaner way to set types?
+    // Question: If the input config is negative, should I warn the user?
+    let config_min = context
+        .config
```

Comment:
The module already contains its config, so you can access it via `module.config`.
https://github.com/starship/starship/blob/master/src/context.rs#L102

Reply:
Hmm, I can't seem to access the config from within the module-generation function though, since it's private to Module.

```
field `config` of struct `module::Module` is private
```

Would adding an accessor to `Module` work? Or should I just make the config `pub`?